<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AptOfflineQtFetch</class>
 <widget class="QDialog" name="AptOfflineQtFetch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>468</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>468</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fetch Packages or Upgrade</string>
  </property>
  <widget class="QLineEdit" name="profileFilePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>46</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFilePathButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>46</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startDownloadButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>165</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/go-down.png</normaloff>:/icons/icons/go-down.png</iconset>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>165</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/dialog-cancel.png</normaloff>:/icons/icons/dialog-cancel.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>16</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the signature file</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="statusProgressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>410</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="progressStatusDescription">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="rawLogHolder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>268</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseZipFileButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>107</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>77</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save archive as</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="zipFilePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>107</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
